tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -42,7 +42,7 @@ public function iSaveAnArticle()
 	public function iShouldSeeTheMessage($message)
 	{
 		$I = $this;
-		$I->waitForText($message, 60, AdminPage::$systemMessageContainer);
```

Comment:
I think we would need to modify all the Wait commands in the whole test suite. For example: https://github.com/joomla-projects/gsoc16_browser-automated-tests/blob/staging/tests/acceptance/administrator/UserCest.php#L36

so any waitForText or waitForElement in all the step files should use the TIMEOUT constant